compiler/src/iree/compiler/Codegen/Common/FoldTensorSubsetIntoVectorTransferOps.cpp

```diff
@@ -268,20 +268,23 @@ class FoldExtractSliceIntoTransferWrite final
     auto init = rewriter.create<tensor::EmptyOp>(
         loc, mixedSizes, extractSliceOp.getType().getElementType());
 
+    auto indices = xferOp.getIndices();
+
     SmallVector<bool> inBounds;
     inBounds.resize(mixedSizes.size());
-    for (auto [idx, vecSize, destSize] :
-         llvm::zip_equal(llvm::seq<int64_t>(0, inBounds.size()),
-                         xferOp.getVectorType().getShape(), mixedSizes)) {
+    for (auto [idx, vecSize, destSize, index] : llvm::zip_equal(
+             llvm::seq<int64_t>(0, inBounds.size()),
+             xferOp.getVectorType().getShape(), mixedSizes, indices)) {
+      inBounds[idx] = false;
       auto maybeCst = getConstantIntValue(destSize);
-      if (!maybeCst) {
-        inBounds[idx] = false;
+      auto maybeIndex = getConstantIntValue(index);
+      if (!maybeCst)
```

Comment:
since this is the only use left of `maybeCst`, maybe inline its definition there and drop the variable. Its name `maybeCst` was undescriptive anyway.

Reply:
Also, we prefer braces around all control flow, particularly when the body contains control-flow statements such as `continue`.